armi/apps.py

```diff
@@ -231,7 +229,7 @@ def getParamRenames(self) -> Dict[str, str]:
         return renames
 
     def registerUserPlugins(self, pluginPaths):
-        """
+        r"""
```

Comment:
@john-science why is the `r` being added? I'm assuming the docs built fine before without it?